src/coinbase/wallet.ts

```diff
@@ -748,6 +749,37 @@ export class Wallet {
     return (await this.getDefaultAddress()).createPayloadSignature(unsignedPayload);
   }
 
+  /**
+   * Creates a Webhook.
```

Comment:
can you add a bit more contents here for the summary

Reply:
done